src/AltinnCore/Common/Configuration/PlatformSettings.cs

```diff
@@ -168,5 +168,21 @@ public string GetApiAuthorizationEndpointHost
                 return Environment.GetEnvironmentVariable("PlatformSettings__ApiAuthorizationEndpointHost") ?? ApiAuthorizationEndpointHost;
             }
         }
+
+        /// <summary>
+        /// Gets or sets the the url for the API PDF
+        /// </summary>
+        public string ApiPdfEndpoint {get; set;}
+
+        /// <summary>
+        /// Gets the url for the API PDF by looking into environment variables
+        /// </summary>
+        public string GetApiPdfEndpoint
+        {
+            get
+            {
+                return Environment.GetEnvironmentVariable("PlatformSettings__ApiPdfEndpoint") ?? ApiPdfEndpoint;
```

Comment:
@DanRJ could the notation be simplified here as well or does that only work i designer?

Reply:
It can be simplified, but test it first